app/controllers/tools_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ToolsController < ApplicationController
+  def index
+    # NO OP
```

Comment:
I would avoid this comment, it doesn't really add anything as empty controller actions for pages that simply render a template are a pretty standard thing in Rails.